django/mgmt/forms.py

```diff
@@ -229,6 +227,14 @@ def cleaned_update_data(self):
         self.update_fields_if_admin()
         return super().cleaned_update_data
 
+    def disable_non_admin_fields(self):
+        """
+        Disable fields that only admins can access.  Use this when providing a
+        create form for the non-admin user.
+        """
+        self.fields['bucket'].disabled = True
+        self.fields['cv_path'].disabled = True
+
```

Comment:
Should disable_non_admin_fields() also disable storage_type?

Reply:
Done.